plugins/native/CMakeLists.txt

```diff
@@ -193,6 +193,14 @@ f3d_plugin_declare_reader(
   FORMAT_DESCRIPTION "Quake 1 MDL model"
 )
 
+f3d_plugin_declare_reader(
+  NAME SPZ
+  EXTENSIONS spz
+  MIMETYPES application/vnd.spz
+  VTK_READER vtkF3DSPZReader
+  FORMAT_DESCRIPTION "Niantic's 3D Gaussians"
```

Comment:
Im not sure thats the correct description for this format but I dont have a better suggestion

Reply:
`Compressed 3D Gaussians`?

Reply:
I don't like it. It's used in Windows file explorer, it should be descriptive for the user.

Reply:
What is used in Windows file explorer ?

Reply:
The `FORMAT_DESCRIPTION` field

Reply:
![image](https://github.com/user-attachments/assets/af901365-5bbb-4432-87d8-4f07903757db)

Reply:
`Compressed 3D gaussian splats` then ?